modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -326,7 +329,7 @@ public CompletableFuture<Void> createZone(DistributionZoneConfigurationParameter
                                     ConfigurationValidationException.class)
                     );
                 } else {
-                    fut.complete(null);
+                    fut.complete(zoneIdContainer[0]);
```

Comment:
You can probably just read the value from the configuration, but ok.
Why not `AtomicInteger` by the way?

Reply:
Just to additionally highlight, for reader, that concurrent access is not a target for this container.